bcda/api/requests.go

```diff
@@ -632,6 +632,7 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 		sinceParam := (!rp.Since.IsZero() || conditions.ReqType == service.RetrieveNewBeneHistData)
 		jobPriority := h.Svc.GetJobPriority(conditions.CMSID, j.ResourceType, sinceParam) // first argument is the CMS ID, not the ACO uuid
 
+		logger.Infof("Adding jobs using %T", h.Enq)
```

Comment:
Sanity check that we are are using the proper queueing library.  I dont totally understand why we seem to have different loggers and different access to them?